### ISSUE TITLE
How to install if `configure` file is missing?

### ISSUE THREAD

**Issue by miguelmorin** [sleuthkit/sleuthkit#2811]
I tried installing from source on macOS. The repo does not have a `configure` file, so I cannot run

```
$ ./configure
```

as mentioned in the [installation instructions](https://github.com/sleuthkit/sleuthkit/blob/develop/INSTALL.txt).

The repo has a `configure.ac`, so following [this advice](https://www.linuxquestions.org/questions/linux-newbie-8/how-to-run-a-configure-ac-file-4175596216/), I first ran `autoconf` and `automake`, then:

```
$ ./configure
configure: error: cannot find required auxiliary files: config.guess config.sub install-sh
```

How can I install TSK from source on macOS?

**Comment by joachimmetz** [comment]
you need to generate `configure` with `./bootstrap` this requires that you have autotools installed (which on MacOS can be tricky). Just stick with the packaged release, if you're not actively developing sleuthkit.

**Comment by miguelmorin** [comment]
Yes, I ended up installing through `ports` and it worked out of the box.
